features/backend/deleting_shipping_methods.feature

```diff
@@ -0,0 +1,18 @@
+@shipping
+Feature: Deleting a shipping method
+  In order to fully manage shipping methods
+  As an Administrator
+  I want to be able to remove shipping methods when they are not used
+
+  Background:
+    Given there is a shipment using "DHL" shipping method
+    And there is a shipping method "DHL Express"
+    And I am logged in as administrator
+
+  Scenario: Not deleting a shipping method which is in use
+    When I delete shipping method "DHL"
+    Then shipping method "DHL" should still exist
```

Comment:
Below one of these 3, we should have:

``And shipping method "DHL Express" should still exist``